oomph_build.py

```diff
@@ -488,7 +488,7 @@ def expanded_path(p):
     ext_group.add_argument("--ext-OOMPH_USE_BOOST_FROM", type=expanded_path, metavar="PATH", help="Use a preinstalled Boost from the given path; used by CGAL.")
 
     # Any additional flags for the external distributions project
-    ext_group.add_argument("--ext-extra-flags", nargs="+", metavar="FLAG", help="Additional raw CMake flags for external_distributions (e.g. -DXYZ=VALUE).")
+    ext_group.add_argument("--ext-extra-cmake-options", nargs="+", metavar="FLAG", help="Additional raw CMake flags for external_distributions (e.g. -DXYZ=VALUE).")
```

Comment:
I always struggle with these overly generic placeholders. Can we specify a real example of an option one may actually want to pass to cmake; ideally something that users are likely to have come across already and will immediately recognise as something that would only be passed to cmake.